src/edit/EditHandle.js

```diff
@@ -35,9 +35,11 @@ L.EditHandle = L.Marker.extend({
 
 	_onHandleDragStart: function() {
 		this._handled.fire('editstart');
+		// this._handled.editing._hideToolbar();
```

Comment:
Just a question, why is this commented out? This shouldn't have any side effects, I guess. Did this introduce bugs?

Reply:
I wrote that! I left it because it was another way of implementing the toolbar UI. This matches the ._showToolbar() onHandleDragEnd, and the result is the toolbar will hide while you're moving any corners and reappear again when you're not. 

I ended up deciding it looks best if the toolbar is just always present. But I think for the sake of future developing and changing things around, I could leave these uncommented lines and put a comment above them explaining the UI changes that will take effect if they are uncommented? Thoughts?

Reply:
@rexagod Ok I updated it with an explanation above it. Will you check it out and lmk if you like this, or its unnecessary and want to get rid of it altogether? 

This library sometimes feels like a maze of figuring out what triggers what so trying to make our lives easier is my thought process here

Reply:
Sure, that'll be perfect! :+1:

Reply:
Cool, this one I'll just leave here because it has no place on the README. The other I made an attempt to move to README and notified you about it below. Resolving this one.

Reply:
I actually tested it and I have updated the code since so that uncommenting these lines does nothing haha. So I deleted them and my commentary.